### PR TITLE
fix: implement dropdown to 'pipeline start from here' on workflow graph

### PR DESCRIPTION
**Pull request by adong**
## Context

Currently, the dropdown selection is only shown at the start of the pipeline, this PR will implement and add dropdown selection to `Start pipeline from here`

## Objective

After
image

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/pipeline-workflow/template.hbs [hunk 1]
```diff
@@ -78,7 +78,7 @@
         {{/if}}
       {{else if (get-length tooltipData.selectedEvent.meta.parameters)}}
         {{#pipeline-parameterized-build
-          buildParameters=tooltipData.selectedEvent.meta.parameters
+          buildParameters=buildParameters
```
Comment (jithine) [inline app/components/pipeline-workflow/template.hbs:81]: does this pull default params, or the event's params from `meta`?

Comment (adong) [reply app/components/pipeline-workflow/template.hbs:81]: Former, default parameters. 

This PR change will get parameters from `pipelines`, not the parameters which were used.

Comment (jithine) [reply app/components/pipeline-workflow/template.hbs:81]: when this parameter popup appears on job restart context it should pick values from the event for the build

File: app/components/pipeline-workflow/component.js [hunk 2]
```diff
@@ -47,6 +48,30 @@ export default Component.extend({
     });
   },
 
+  buildParameters: computed('tooltipData', function preselectBuildParameters() {
+    const defaultParameters = this.getWithDefault('pipeline.parameters', {});
+    const buildParameters = copy(defaultParameters, true);
+
+    if (this.tooltipData) {
```
Comment (adong) [inline app/components/pipeline-workflow/component.js:55]: I just updated that it will show the used value as the selected value. @jithine

**Pull request merged**